cloud/pkg/router/provider/eventbus/eventbus.go

```diff
@@ -112,18 +112,30 @@ func (*EventBus) Forward(target provider.Target, data interface{}) (response int
 
 func (eb *EventBus) GoToTarget(data map[string]interface{}, stop chan struct{}) (interface{}, error) {
 	messageID, ok := data["messageID"].(string)
-	body, ok := data["data"].([]byte)
-	param, ok := data["param"].(string)
+	if !ok {
+		err := errors.New("data transform failed, messageID must be not null")
```

Comment:
better to print type is not matched, !ok happens when  maybe value is nil or type does not match string

Reply:
Ok, fixed